podcasts/End of Year/Stories/YearOverYearStory.swift

```diff
@@ -0,0 +1,211 @@
+import SwiftUI
+import PocketCastsServer
+import PocketCastsDataModel
+
+struct YearOverYearStory: ShareableStory {
+    var duration: TimeInterval = 5.seconds
+
+    let identifier: String = "year_over_year"
+
+    let yearOverYearListeningTime: YearOverYearListeningTime
```

Comment:
Can we rename this to just `data` since it's already relative to the story which is named year over year?


```suggestion
    let data: YearOverYearListeningTime
```

Reply:
Done in 4b40128a13db3db5b98e0a3b35eb1f9444c7ffc6